custom-recipes/nlp-analysis-ontology-tagging/recipe.json

```diff
@@ -392,15 +392,15 @@
             "selectChoices": [
                 {
                     "value": "one_row_per_match",
-                    "label": "row per match"
+                    "label": "one row per match"
```

Comment:
Nitpick: would look better with capitalize "One row per match"
(and same for other labels)

Reply:
alright ! 9cd82af